app/javascript/components/MockItems.js

```diff
@@ -14,6 +15,7 @@ const items = [
     party_id: 1,
     user_id: 2,
     first_name: 'Dr',
+    last_name: 'Pepper',
```

Comment:
It seems as though mockItems should not have first_name or last_name properties

Reply:
It also seems that that the show method on items controller would have a property of `user` on which you could access the first_name and last_name properties on a user object

Reply:
We did this so that we can make sure we can view the first name and last name in front end, however this functionality will change once we start doing fetch calls.  The show method in items controller is the building block to achieve this when we start doing our fetch calls, however there will be some refactoring to make this happen together in front and backend.